retrofit/src/main/java/retrofit/OkHttpCall.java

```diff
@@ -157,4 +158,24 @@ public void cancel() {
       rawCall.cancel();
     }
   }
+
+  @Override
+  public String toString() {
+    return "OkHttpCall{"
+        + "retrofit="
+        + retrofit
+        + ", requestFactory="
+        + requestFactory
+        + ", responseConverter="
+        + responseConverter
+        + ", args="
+        + Arrays.toString(args)
```

Comment:
Yeah… Similar to the WebSockets threads naming issue in the OkHttp.

>By moving its string value into an exception it could be inadvertently logged
I've removed `toString()` and `addValueAsLastCause()` for now.

But it won't help with untrusted code since it could use reflection to retrieve fields! (though reflection can be disabled).